File: src/tests/Behavioral/ReflectFieldMetadata/fieldlib/fieldlib.cs
namespace go.ReflectFieldMetadata;

partial class fieldlib_package {

[GoType] partial struct Outer {
    public nint Exported;
    internal nint unexported;
}

public static nint Sum(this Outer o) {
    return o.Exported + o.unexported;
}

} // end fieldlib_package

File: src/tests/Behavioral/ReflectFieldMetadata/fieldlib/fieldlib.go
// fieldlib is the FOREIGN package for ReflectFieldMetadata's PkgPath row: a defined type in the
// parent over Outer must report this package for the unexported field it did not declare.
package fieldlib

type Outer struct {
	Exported   int
	unexported int
}

// Keep the unexported field referenced so the package has a use for it.
func (o Outer) Sum() int { return o.Exported + o.unexported }
